src/main/resources/assets/wotr/shaders/rift_portal.json

```diff
@@ -0,0 +1,97 @@
+{
+  "vertex": "wotr:rift_portal",
+  "fragment": "wotr:rift_portal",
+  "defines": {
+    "values": {
+      "ALPHA_CUTOUT": "0.1"
+    }
+  },
+  "samplers": [
+    {
+      "name": "Sampler0"
+    },
+    {
+      "name": "Sampler1"
+    }
+  ],
+  "uniforms": [
+    {
+      "name": "ModelViewMat",
+      "type": "matrix4x4",
+      "count": 16,
+      "values": [
```

Comment:
Overly zealous json formatter?

Reply:
Probably!

Reply:
Yeah, the IntelliJ formatter places a value per line. I'm not inclined to fight it. The value is just a default, Minecraft overwrites it with an actual value during use.

Reply:
I've updated it, but will have to see if it sticks or gets changed on accident in the future.